src/Apps/Auction/getRedirect.ts

```diff
@@ -35,14 +35,22 @@ export function registerRedirect(
   return null
 }
 
-export function confirmBidRedirect(
-  artwork: routes_BidQueryResponse["artwork"],
-  me: routes_BidQueryResponse["me"]
-): Redirect | null {
+export function confirmBidRedirect({
```

Comment:
Should we pass in location here, and redirect to the current URL, rather than the `bidPath`?

Reply:
I would love for us to adopt a redirect structure similar to what's in the OrderApp today: https://github.com/artsy/reaction/blob/master/src/Apps/Order/redirects.tsx#L163-L174

I love the path matching and explicitly named redirect rules. :heart: 

This certainly looks clear enough for now though!

Reply:
> Should we pass in location here, and redirect to the current URL, rather than the bidPath?

To clarify: and redirect to the log_in page with the `redirect_uri` being equal to the current path? If so, then yes, I think that's more robust.